OuterWildsRandomSpeedrun/OuterWildsRandomSpeedrun.cs

```diff
@@ -64,9 +65,23 @@ private void Update()
             if (_isGameStarted && _shouldWarp)
             {
                 HandleBasicWarp();
+                ShowTimerText();
             }
         }
 
+        private void ShowTimerText()
+        {
+            var screenPromptListObj = GameObject.Find("ScreenPromptListBottomLeft");
+            var screenPromptList = screenPromptListObj.GetComponent<ScreenPromptList>();
+            
+            var screenPrompt = new ScreenPrompt("");
+            screenPrompt.SetText($"<color={OW_ORANGE_COLOR}>01:02.235</color>");
```

Comment:
any concern with this overwriting existing text? i think this spot is where the launch codes and eye coordinates live on screen.

Reply:
ScreenPromptList is a list of ScreenPrompts; the AddScreenPrompt() method inserts a new one into the beginning of the list. I verified using the Unity explorer mod that I can still activate the other prompts that exist in the list and have them play nicely.

Reply:
launch codes aren't _super_ important but i could see it being confusing if the player starts on timber hearth and doesn't see them. i'm more concerned about the eye coordinates being missing, since that's a legitimately useful reference that's no longer accessible to the player.

Reply:
oh great! thanks for checking. this looks good then